source/github-helpers/get-comment-author.ts

```diff
@@ -19,8 +19,8 @@ Note: Bots are used as `name[bot]`, `app/name`, or `apps/name` depending on the
 */
 export default function getCommentAuthor(anyElementInsideComment: Element): string {
 	const avatar: HTMLImageElement = anyElementInsideComment
-		.closest(['.TimelineItem', '.review-comment'])!
-		.querySelector(['.TimelineItem-avatar img', 'img.avatar'])!;
+		.closest(['.TimelineItem', '.review-comment', '.react-issue-comment'])!
+		.querySelector(['.TimelineItem-avatar img', 'img.avatar', 'img[data-testid=github-avatar]'])!;
```

Comment:
These should probably be placed on multiple lines and commented. I assume that they're, in order:

- PR comments (and pre-issue redesign issue comments)
- PR review comments
- Issue comments

So they should all be commented this way